docs/application/native/guides/connectivity/connectivity-cover-n.md

```diff
@@ -22,6 +22,9 @@ You can connect to a Wireless Local Area Network (WLAN) and transfer data over t
 - [Wi-Fi Direct&reg;](wifi-direct-n.md) **in mobile applications only**
 You can find nearby Wi-Fi Direct devices and discover their details. You can also form a Wi-Fi Direct group to communicate over a peer-to-peer link without wireless access points (base stations) in the infrastructure mode.
 
+- [STC (Smart Traffic Control)](stc-n.md)
+You can fetch statistics about network data consumption by system and applications.
+
 - [Telephony Information](telephony-n.md)
 You can access various information from the Telephony Service. You can access details about the voice and video call states, currently used SIM card, and modem. You can also retrieve general details about the telephony network the device is currently connected to.
```

Comment:
1) Access various information about the voice and voice call status that currently uses SIM card and Modem from the Telephony Service. 
2) Retrieving general details about the telephony network from the currently connected device.